contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+If you confirm, the existing chainquery and chainquery-mysql deployments will be
+deleted, and pods will be terminated, ***and the contents of the Persistent
+Volume Claim (PVC) for chainquery-mysql will be deleted.*** The snapshot will be
+downloaded and restored in its place.
+
+Once the snapshot is restored, upgrade the release to restore the chainquery and
+chainquery-mysql deployments, and restart pods:
+
+```
+k8s-lbry upgrade
+```
+
+You can verify that the database now has data up to the height of the database
+snapshot. Login to the mysql shell:
+
+```
+k8s-lbry chainquery-mysql-client
+```
+
+Then query for the latest block height:
+
+```
+mysql> select height from chainquery.block order by id desc limit 1;
++--------+
+| height |
++--------+
+| 561080 |
++--------+
+1 row in set (0.00 sec)
+```
+
+Also verify that chainquery is again happy. View the chainquery logs:
+
+```
+k8s-lbry logs chainquery
+```
+
+Press Ctrl-C to quit viewing the logs.
+
+### lbrynet API service (not for spee.ch)
+
+This is for a standalone lbrynet API service inside your cluster. Blob storage
+goes to its own persistent volume, but is configured with `save_files=false`.
+There is no outside access to the Downloads directory provided. You can stream
+blobs from lbrynet via `http://lbrynet:5279/get/CLAIM_NAME/CLAIM_ID`.
+
+This particular lbrynet configuration won't work for spee.ch (v0.5.12). spee.ch
+needs direct access to the Downloads directory of lbrynet. **If you are wanting
+lbrynet for spee.ch, skip this section, and head directly to the [spee.ch
+section](#speech-and-lbrynet-sidecar-and-mysql), which implements its own
+lbrynet sidecar.**
+
+Edit `values-dev.yaml` and set `lbrynet.enabled` to `true`.
+
+Upgrade the release to turn on lbrynet:
+
+```
+k8s-lbry upgrade
+```
+
+You can view the lbrynet logs:
+
+```
+k8s-lbry logs lbrynet
+```
+
+#### IMPORTANT - Backup your cluster wallet
+
+The wallet is created inside the `lbrynet` persistent volume.
+
+Copy the wallet in case the volume gets destroyed:
+
+```
+k8s-lbry lbrynet-copy-wallet /tmp/k8s-lbry-lbrynet-wallet-backup.json
+```
+
+Check the contents of `/tmp/k8s-lbry-lbrynet-wallet-backup.json` and move the
+file to a safe place for backup (make sure to delete the temporary file.)
+
+Once your wallet is backed up, you can generate a receiving address in order to
+deposit LBC:
+
+```
+k8s-lbry lbrynet address unused
+```
+
+### spee.ch (and lbrynet sidecar and mysql)
+
+*Note: Throughout this deployment, the unstylized name `speech` is used.*
+
+Speech needs three containers, running in two pods:
+
+ * `speech` pod:
+ 
+   * speech, the nodejs server container.
+
+   * lbrynet, running in the same pod as speech, so as to share one downloads
+     directory. (This is called a 'sidecar' container, which is guaranteed to
+     run on the same kubernetes node as the spee.ch container.)
+   
+ * `speech-mysql` pod:
+ 
+   * mysql for storing the speech database.
+
+Edit `values-dev.yaml`. 
```

Comment:
Can you update references to these files with URL's referencing their location in the repository?  Just might help people know where the defaults are.

Reply:
values-dev.yaml does not exist in the repository, only values.yaml does. values-dev is created by run.sh when the user runs setup.